frontend/src/lib/api.ts

```diff
@@ -2241,6 +2257,39 @@ const api = {
         },
     },
 
+    schemaPropertyGroups: {
+        async list(projectId?: ProjectType['id']): Promise<{ results: any[] }> {
+            return new ApiRequest().schemaPropertyGroups(projectId).get()
+        },
+        async get(id: string, projectId?: ProjectType['id']): Promise<any> {
+            return new ApiRequest().schemaPropertyGroupsDetail(id, projectId).get()
+        },
+        async create(data: any, projectId?: ProjectType['id']): Promise<any> {
```

Comment:
Nit: these functions can use the types in `schemaManagementLogic.ts` right? 
e.g.
```
async create(data: Partial<SchemaPropertyGroup>, projectId?: ProjectType['id']): Promise<SchemaPropertyGroup> { ...

```